README.md

```diff
@@ -110,7 +110,7 @@ def mystery6(s)
 end
 ```
 
-- What is mystery6("goodnight moon")?
-- What is mystery6("Ada Developers Academy")?
-- What is mystery6("Hi, there!")?
+- What is mystery6("goodnight moon")? moon goodnight
+- What is mystery6("Ada Developers Academy")? Academy Developers Ada
+- What is mystery6("Hi, there!")? there, Hi!
```

Comment:
Here we should have quotations as well and this also shows _why_ we should do it because in these answers there is a leading space on all, but you can't tell that if there aren't quotations. i.e. answers should be " moon goodnight", " Academy Developers Ada",  and " there, Hi!"